File: Beecrowd/for_loop.py
#For loop
#repita 10 vezes
for i in range(1,11):
    #mostre o contador
    print("i=", i)

import random

s = 0

for i in range(0,10001):
    n = random.randint(1, 10001)
    s = s + n
    print(n)

m = s / 10000

print('média:', m)

File: Códigos de aula/for_loop.c
//inclusão da biblioteca de entrada e saída
#include <stdio.h>

//ponto de início do programa
int main(){
	
	//repita 10 vezes
	for (int i=1; i <= 10; i++){
		//mostre o contador de repetições
		printf("i = %d\n", i);
	}
	
	//fim do programa
		return 0;
}
///////////////////////////////////////////////////////////////////////

#include <stdio.h>
#include <time.h>
#include <stdlib.h>


int main(){
    srand(time(NULL));
    int s = 0;
    for (int i=1; i <= 10001; i++){
        int r = rand() % 10001;
        printf("%d ", r);
        s = s + r;
    }    
    int m = s / 10001;

    printf("\n média é %d", m);
}
